### ISSUE TITLE
Is it possible to change the workspace-resource-id after creating a cluster?

### ISSUE THREAD

**Issue by mkfsn** [Azure/AKS#878]
**What happened**:

I created some aks clusters without specifying the workspace-resource-id when creating them, I am wondering if that is possible to change the OMS workspace of that cluster? As I need to scale the cluster up/down and I don't want to disconnect from the default workspace and connect to the workspace I want for every VMs.

**What you expected to happen**:

Maybe I can update the cluster's workspace using "az aks ..."? I couldn't find a solution from the documentation.

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

I know I can go to "Log Analytics" > Select my workspace > "Virtual machines", and then click on every VM I want to disconnect from the other workspace and connect to the current workspace, but I would like to know if that is also possible to modify all the VM in that cluster to a specified OMS workspace using a script or somewhere in portal.azure.com.

**Environment**:
- Kubernetes version (use `kubectl version`):
Client Version v1.10.5

- Size of cluster (how many worker nodes are in the cluster?)
1~20

- General description of workloads in the cluster (e.g. HTTP microservices, Java app, Ruby on Rails, machine learning, etc.)
HTTP microservices

- Others:

To sum up, basically two questsions:
- Is it possible to change the OMS workspace of a aks cluster?
- Is it possible to change the OMS workspace of VMs in a aks cluster in one time rather than configure them one-by-one?

**Comment by mgrabarz** [comment]
I would personally use following azure cli commands:

az aks disable-addons -a monitoring -n MyExistingManagedCluster -g MyExistingManagedClusterRG
az aks enable-addons -a monitoring -n MyExistingManagedCluster -g MyExistingManagedClusterRG --workspace-resource-id <ExistingWorkspaceResourceID>

**Comment by jluk** [comment]
Closing as the update path for addons does enable swapping the workspace

**Comment by mohoff** [comment]
@jluk What do you mean by "update path" ? `az aks update-addons` does not seem to exist.

**Comment by jluk** [comment]
I am referring to the previous comment. This is the path to update an addon today. @vishy for feedback on OMS documentation though.

> I would personally use following azure cli commands:
> 
> az aks disable-addons -a monitoring -n MyExistingManagedCluster -g MyExistingManagedClusterRG
> az aks enable-addons -a monitoring -n MyExistingManagedCluster -g MyExistingManagedClusterRG --workspace-resource-id ExistingWorkspaceResourceID
